all/pom.xml

```diff
@@ -420,6 +420,11 @@
             <artifactId>mysql-connector-java</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
```

Comment:
remove dependency. There is no direct dependency on ojdbc in the code. If the business uses oracle, it must be dependent on ojdbc in his pom.